src/math_utils.rs

```diff
@@ -185,7 +194,24 @@ mod tests {
             bigint_str!(
                 b"1545825591488572374291664030703937603499513742109806697511239542787093258962"
             ),
-            div_mod(a, b, prime)
+            div_mod(a, b, &prime)
+        );
+    }
+
+    #[test]
+    fn compute_safe_div() {
+        let x = bigint!(26);
+        let y = bigint!(13);
+        assert_eq!(safe_div(&x, &y), Ok(bigint!(2)));
+    }
+
+    #[test]
+    fn compute_safe_div_fail() {
+        let x = bigint!(25);
+        let y = bigint!(4);
+        assert_eq!(
+            safe_div(&x, &y),
+            Err(VirtualMachineError::SafeDivFail(bigint!(25), bigint!(4)))
```

Comment:
Add a test case for division by zero.

Reply:
I can't because the divisor is always N

Reply:
partially resolved:
https://github.com/lambdaclass/cleopatra_cairo/pull/333/commits/bff8fe8a880eec8430a711e8508ccbaee3da8247